csob/payment.py

```diff
@@ -0,0 +1,39 @@
+from decimal import Decimal
+from typing import Optional, Union
+
+
+class Item:
```

Comment:
itemů tam jde posílat více?

Reply:
@matllubos Jde tam poslat 1 nebo 2 maximálně s tím že je doporučené jedno mít jako objednávku a druhé jako shipping nebo něco tak

Reply:
ok tedy to nemá nic společného s order itemem, to je pak ta quantity pěkná ptákovina :D

Reply:
a ještě je ve specifikaci že když je `quantity=0` tak se tam ukáže u té položky "Zdarma" :D nevím co tím měl uator v plánu :D